src/main/java/tech/artisanhub/ARFFGenerator/CSVtoARFF.java

```diff
@@ -11,19 +11,24 @@
     private static final String TRAINING_DATA = "/home/rnavagamuwa/Documents/CSE/FYP/Datasets/DEBS-2013-SoccerField/result_1st_goal.txt";
     private static final String OUTPUT_DATA = "/home/rnavagamuwa/Documents/CSE/FYP/Datasets/DEBS-2013-SoccerField/ARFF_result_1st_goal.arff";
 
+
+
+
     public static void main(String[] args) throws Exception {
 
+        String salihPath = "/home/jawadhsr/Desktop/FYP/FIles/iris_replacedNamedWithInts.csv";
         // load CSV
         CSVLoader loader = new CSVLoader();
-        loader.setSource(new File(TRAINING_DATA));
+        loader.setSource(new File(salihPath));
         Instances data = loader.getDataSet();
         System.out.println("Source CSV has been successfully loaded");
 
+        String salihOutput = "/home/jawadhsr/Desktop/FYP/FIles/output.arff";
         // save ARFF
         ArffSaver saver = new ArffSaver();
         saver.setInstances(data);
-        saver.setFile(new File(OUTPUT_DATA));
-        saver.setDestination(new File(OUTPUT_DATA));
+        saver.setFile(new File(salihOutput));
+        saver.setDestination(new File(salihOutput));
         saver.writeBatch();
         System.out.println("ARFF has been successfully generated");
```

Comment:
@Jawadh-Salih machan remove this file from the commit. This is not actually a change of code. Will be very hard to track if something goes wrong with  these kind of commits :smile: